README.md

```diff
@@ -1,2 +1,107 @@
+<img src="http://www.braveterry.com/wp-content/uploads/2015/03/swagger2.png"/>
+
 # cowboy-swagger
-Swagger integration for Cowboy (built on trails)
+[Swagger](http://swagger.io/) integration for [Cowboy](https://github.com/ninenines/cowboy) (built on [trails](https://github.com/inaka/cowboy-trails)).
+
+## Contact Us
+For **questions** or **general comments** regarding the use of this library,
+please use our public [hipchat room](https://www.hipchat.com/gpBpW3SsT).
+
+If you find any **bugs** or have a **problem** while using this library, please
+[open an issue](https://github.com/inaka/elvis/issues/new) in this repo
+(or a pull request :)).
+
+And you can check all of our open-source projects at [inaka.github.io](http://inaka.github.io).
+
+## Why Cowboy Swagger?
+Simple, because there isn't a tool in Erlang to document Cowboy RESTful APIs, easy and fast,
+improving development productivity.
+
+With `cowboy_swagger` is possible to integrate Swagger to your Erlang projects, that use Cowboy as web server.
+It is extremely easy to use, and with just a few steps you'll have a nice Web documentation for your RESTful APIs.
+
+To learn a bit more about Swagger, please check this [blog post](http://inaka.net/blog/2015/06/23/erlang-swagger-2015/).
+
+## How to Use it?
+This is the best part, is extremely easy.
+
+### 1. Document each Cowboy Handler
+Because `cowboy_swagger` runs on top of `trails`, the first thing that you have to do
+is document all about you handler within the trails metadata. Keep in mind that
+all fields defined within each method into the metadata must be compliant with the
+[Swagger specification](http://swagger.io/specification).
+
+For example, suppose that you have `example_echo_handler`, so it must implement the `trails/0`
+callback from `trails_handler` behavior:
```

Comment:
behavior (American) behaviour (British). Choose the one according to your writing style, but keep in mind that it applies for everything, you can't mix words from one and the other in the same text.